data_structures/avltree.cpp

```diff
@@ -10,27 +10,28 @@
 #include <iostream>
```

Comment:
Explain why each header is required

Reply:
Done in 5ce6bf01571baa833280f8823d23454f2c4bf9ff.